test/e2e/portforward.go

```diff
@@ -77,7 +81,52 @@ func pfPod(expectedClientData, chunks, chunkSize, chunkIntervalMillis string) *a
 	}
 }
 
-func runPortForward(ns, podName string, port int) (*exec.Cmd, int) {
+type portForwardContext struct {
+	cmd    *exec.Cmd
+	port   int
+	stdout io.Closer
+	stderr io.Closer
```

Comment:
No, they're pipes that come from https://github.com/ncdc/kubernetes/blob/go16-portforward-e2e/test/e2e/portforward.go#L135, https://github.com/ncdc/kubernetes/blob/go16-portforward-e2e/test/e2e/util.go#L1638.